index.js

```diff
@@ -652,4 +657,18 @@ HtmlWebpackPlugin.prototype.applyPluginsAsyncWaterfall = function (compilation)
   };
 };
 
+/**
+ * Get all chunks from compilation
+ * @param compilation
+ * @return {Array}
+ */
+HtmlWebpackPlugin.prototype.getAllChunks = function (compilation) {
```

Comment:
I don't think it should returns only the `chunks`, it should return the whole JSON so we can use it for https://github.com/jantimon/html-webpack-plugin/blob/aedc623c770bce51f460da6a154690064fba2417/index.js#L255

Reply:
Push new opt commit, these two places are handled uniformly